convene-web/app/javascript/src/identity.js

```diff
@@ -0,0 +1,18 @@
+export default class Identity {
+  set name(name) {
+    sessionStorage.setItem("name", name);
+    return true;
+  }
+
+  set pronoun(pronoun) {
+    sessionStorage.setItem("pronoun", pronoun);
+    return true;
+  }
+
+  get personInfo() {
+    return {
+      name: sessionStorage.getItem("name"),
+      pronoun: sessionStorage.getItem("pronoun"),
+    };
+  }
```

Comment:
What advantages does the `personItem` method provide over a `get name() { ... }` that encapsulates the sessionStorage call?

Reply:
I think `get name()` makes more sense.